qemu/tests/balloon_stress.py

```diff
@@ -38,15 +41,19 @@ def run_video():
     timeout = float(params.get("login_timeout", 360))
     session = vm.wait_for_login(timeout=timeout)
 
-    driver_name = params["driver_name"]
-    utils_test.qemu.setup_win_driver_verifier(driver_name, vm, timeout)
+    if params['os_type'] == 'windows':
+        driver_name = params["driver_name"]
+        utils_test.qemu.setup_win_driver_verifier(driver_name, vm, timeout)
 
-    error_context.context("Run video background", logging.info)
-    bg = utils_misc.InterruptedThread(run_video)
+        balloon_test = BallooningTestWin(test, params, env)
+    else:
+        balloon_test = BallooningTestLinux(test, params, env)
+
+    error_context.context("Run stress background", logging.info)
+    bg = utils_misc.InterruptedThread(run_stress, (test, params, env, vm))
     bg.start()
 
-    repeat_times = int(params.get("repeat_times", 500))
-    balloon_test = BallooningTestWin(test, params, env)
+    repeat_times = int(params.get("repeat_times", 1000))
```

Comment:
500 is requested for windows, it's better to use 500 instead of 1000

Reply:
This change is agreed with @huangyum and @lijinlijin , Thanks!